util/repr-bytes/src/lib.rs

```diff
@@ -434,9 +433,28 @@ macro_rules! derive_debug_and_display_hex_from_as_ref {
         }
 
         impl core::fmt::Display for $mytype {
+            fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+                write!(f, "{:x}", self)
+            }
+        }
+
+        impl core::fmt::LowerHex for $mytype {
             fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
                 let data: &$asref = self.as_ref();
-                write!(f, "{}", $crate::_exports::hex_fmt::HexFmt(data))
```

Comment:
Why not use `HexFmt`? It's `no_std`

Reply:
i need to double check but, iirc i had problems with it pulling in `bytes` which was upset about a lack of a particular platform atomic for one of the targets... so it was easier just to turn it off / remove the dependency.